ts/util/Retries.ts

```diff
@@ -26,7 +26,11 @@
  *  The legacy MathJax object  (FIXME: remove this after all v2 code is gone)
  */
 
-declare const MathJax: { Callback: { After: Function } };
+declare const MathJax: {
+  Callback: {
+    After: (cb1: () => void, cb2: () => void) => Promise<any>;
```

Comment:
This isn't actually the correct argument list.  A callback can be `Function`, `[Function]`, `[Function, ...args: any[]]`, `[Object, Function]`, `[Object, Function, ...args: any[]]`, `[string, Object]`, `[string, Object, ..args: any[]]`, or `{hook: Function, data: any[], object: Object}`, and the arguments to `After()` is a list of these, potentially more than 2.

But I guess this is sufficient for the usage here.

Reply:
Indeed, but since we are never actually using it in the code base, I thought I go for the *arbitrary function* type as discussed in https://www.typescriptlang.org/docs/handbook/2/functions.html#function